enclave-tls/rules/sgx_env.mk

```diff
@@ -112,7 +112,7 @@ else
   sgx_common_flags += -O2
 endif
 sgx_common_flags += \
-  -DSGX_ENCLAVE -I$(SGX_SDK)/include \
+  -I$(SGX_SDK)/include \
```

Comment:
SGX_ENCLAVE should be a common flag used to identify current build context, and here is a good place to define it.